### DISCUSSION TITLE
How to directly obtain site fraction of sublattice

### DISCUSSION THREAD

**Discussion by MWW726** [pycalphad/pycalphad#379] [Q&A] [answered]
What I need is site fraction at various temperatures and compositions,  I want site fraction to be  output. How to do. Please

**Comment by richardotis** [comment, marked as answer]
For an equilibrium calculation, `eq = equilibrium(...)`, the object `eq.Y` contains the equilibrium values of the site fractions. The site fractions will be in sublattice order, and then in alphabetical order (by component) within each sublattice.

Details in the documentation: https://pycalphad.org/docs/latest/examples/UsingCalculationResults.html#equilibrium-results

**Reply by MWW726** [reply]
OK, THANKS

**Comment by bocklund** [comment]
There's now also an example notebook in our [cookbook](https://github.com/pycalphad/cookbook) that gives an example for extracting site fractions vs. temperature to see an order/disorder transformation: https://github.com/pycalphad/cookbook/blob/master/OrderingSiteFractions/OrderingSiteFractions.ipynb
